pkg/reward/reward.go

```diff
@@ -70,7 +71,7 @@ type Reward struct {
 }
 
 // Current returns the profit in percents based on the current TRB price.
-func (self *Reward) Current(ctx context.Context, slot *big.Int, gasPriceEth1e18 *big.Int, client contracts.ETHClient, account *eth.Account) (int64, error) {
+func (self *Reward) Current(ctx context.Context, slot *big.Int, gasPriceEth1e18 *big.Int, client *ethclient.Client, account *eth.Account) (int64, error) {
```

Comment:
the client is not used, is this a mistake?

Reply:
I used it in line#123, but i can remove it as we have the gas price argument.

Reply:
Oh, actually I used it to call `EstimateGas` method!